### PR TITLE
[IMP] rma: auto send confirmation email

### PR DESCRIPTION
**Pull request by chienandalu**
Now we can set if we want to send the mail notification automatically to the RMA customer

cc @Tecnativa TT26371

### CODE REVIEW

File: rma/models/rma.py [hunk 1]
```diff
@@ -528,10 +528,23 @@ def unlink(self):
...
 
+    def _send_confirmation_email(self):
+        """Auto send notifications"""
+        for rma in self.filtered(lambda p: p.company_id.send_rma_confirmation):
+            rma_template_id = (
+                rma.company_id.rma_mail_confirmation_template_id.id
+            )
+            rma.with_context(
+                force_send=True, 
+                mark_rma_as_sent=True
```
Comment (pedrobaeza) [inline rma/models/rma.py:539]: Why this context?

**Review by pedrobaeza — COMMENTED** [review]
Check Travis

File: rma/models/rma.py (hunk 1 above)
Comment (chienandalu) [reply rma/models/rma.py:539]: It changes the state of the RMA `sent` field when it passes by `message_post`: https://github.com/OCA/rma/blob/12.0/rma/models/rma.py#L1144

**Comment by chienandalu** [comment]
Fixed

**Comment by chienandalu** [comment]
This is ready

**Review by marcelsavegnago — APPROVED** [review]
LGTM

**Pull request merged**